Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/CompositingCoordinator.cpp

```diff
@@ -212,14 +209,8 @@ bool CompositingCoordinator::flushPendingLayerChanges(OptionSet<FinalizeRenderin
             });
 
         m_client.commitSceneState(m_nicosia.scene);
-#if !HAVE(DISPLAY_LINK)
         m_forceFrameSync = false;
-#endif
     }
-#if HAVE(DISPLAY_LINK)
-    else
-        m_client.commitSceneState(nullptr);
-#endif
```

Comment:
Isn't this commitSceneState removal breaking canvas and webgl animations?

Reply:
Yes, indeed. I'll try to fix it